lib/segment/src/data_types/vectors.rs

```diff
@@ -68,7 +68,7 @@ impl<'a> TryFrom<VectorRef<'a>> for &'a MultiDenseVector {
 
     fn try_from(value: VectorRef<'a>) -> Result<Self, Self::Error> {
```

Comment:
I propose to remove this method. We have `TypedMultiDenseVectorRef<f32>` which replace `&MultiDenseVector` and can be converted from `VectorRef::Dense`